contracts/contracts/strategies/CurvePoolBoosterDirect.sol

```diff
@@ -0,0 +1,259 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Initializable } from "../utils/Initializable.sol";
+import { Strategizable } from "../governance/Strategizable.sol";
+import { IVotemarket } from "../interfaces/IVotemarket.sol";
+
+/// @title CurvePoolBooster
+/// @author Origin Protocol
+/// @notice Contract to manage interactions with VotemarketV2 for a dedicated Curve pool/gauge.
+contract CurvePoolBoosterDirect is Initializable, Strategizable {
```

Comment:
nit: we should probably move these contracts into their own directories. They're not Vault strategies, so don't really belong here

Reply:
Any suggestion for the the directory name?

Reply:
I ain't sure. But since we are naming the contracts "PoolBooster", we can probably use the same as the name of the directory as well